### PR TITLE
Import `getSlug` instead of using `this.getSlug`

### PR DESCRIPTION
**Pull request by pmn4**
Impact: **minor**  
Type: **bugfix**

## Issue
`this.getSlug` is used in the Reaction core (server-side) file, however, `Reaction.getSlug` is defined outside of the file (in index.js). This is ticky-tack because the code is working everywhere it used, but I feel this is a more correct implementation.
This came up when writing a unit test for `getPrimaryShopPrefix`.

## Solution
import `getSlug` instead of relying on `this.getSlug` which could potentially be undefined.

## Testing
You will not notice a difference in practice because `this.getSlug` is defined by the time it is used, but running the test from this PR should highlight the issue.

### CODE REVIEW

File: imports/plugins/core/core/server/Reaction/core.app-test.js [hunk 1]
```diff
@@ -155,6 +155,16 @@ describe("Server/API/Core", () => {
     });
   });
 
+  describe.only("#getPrimaryShopPrefix", () => {
```
Comment (aldeed) [inline imports/plugins/core/core/server/Reaction/core.app-test.js]: @pmn4 Remove `.only`

**Comment by aldeed** [comment]
Looks good other than the .only

**Comment by pmn4** [comment]
ah! I purged my local repo recently and my pre-push hooks went with it. I'll take care of it tonight

**Comment by aldeed** [comment]
@pmn4 Reminder

**Pull request merged**